test/e2e/gameserver_test.go

```diff
@@ -491,13 +491,16 @@ func TestGameServerReserve(t *testing.T) {
 	assert.Equal(t, "ACK: RESERVE\n", reply)
 
 	gs, err = framework.WaitForGameServerState(readyGs, agonesv1.GameServerStateReserved, time.Minute)
-	assert.NoError(t, err)
-	assert.Equal(t, agonesv1.GameServerStateReserved, gs.Status.State)
+	assert.NoError(t, err, readyGs.ObjectMeta.Name)
+	assert.Equal(t, agonesv1.GameServerStateReserved, gs.Status.State, readyGs.ObjectMeta.Name)
 
 	// it should go back after 10 seconds
 	gs, err = framework.WaitForGameServerState(readyGs, agonesv1.GameServerStateReady, 15*time.Second)
 	assert.NoError(t, err)
 	assert.Equal(t, agonesv1.GameServerStateReady, gs.Status.State)
+
+	logrus.Info("SLEEPING")
+	time.Sleep(time.Minute)
```

Comment:
Was the sleep for local development? I don't think we want to commit that.

Reply:
:man_facepalming: